examples/AspNetCore/Startup.cs

```diff
@@ -23,6 +23,8 @@
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
 using Microsoft.OpenApi.Models;
+using OpenTelemetry;
+using OpenTelemetry.Resources;
```

Comment:
I had to add both `OpenTelemetry` and `OpenTelemetry.Resources` here because the `AddService` extension method is in the `OpenTelemetry` namespace. Was this intentional? Relates to #1541 in that I think we should aim to be consistent.

Reply:
this requires some changes. Lets merge this PR, and follow up to make it consistent.

Reply:
@alanwest @cijothomas I put the extensions into `OpenTelemetry` namespace instead of `OpenTelemetry.Resources` so that people could use the `TracerBuilderExtension.ConfigureResource` extension without an additional namespace having to be imported. But that ended up being removed so we can probably move into `OpenTelemetry.Resources` without any issues, people will just need `OpenTelemetry.Trace` & `OpenTelemetry.Resources` in their startup usings (if they want to use `ResourceBuilder`). Or we put all the builders into `OpenTelemetry` namespace so people only need that single using in their startup? 🤷

Reply:
Yes we discussed this in todays SiG meeting as well.. Lets address the overall namespace changes separate from this PR.

Reply:
Ok, good to know the history here. We can continue the namespace conversation separate from this PR.

Reply:
Doh, it backfired on me! Just updated some code to RC1 and I had to _add_ using OpenTelemetry because of this. I ended up with:
```csharp
using OpenTelemetry;
using OpenTelemetry.Resources;
using OpenTelemetry.Trace;
```
All being required to configure TracerProvider w/ Resources in startup code. That sucks. Sorry!